doc/topics/releases/nitrogen.rst

```diff
@@ -61,6 +61,49 @@ State Module Changes
   start/stop the service using the ``--no-block`` flag in the ``systemctl``
   command. On non-systemd minions, a warning will be issued.
 
+- The :py:func:`module.run <salt.states.module.run>` state dropped its previous
+  syntax with ``m_`` prefix for reserved keywords. Additionally, it allows
+  running several functions in a batch.
+  **NOTE: you need explicitly turn on new behaviour (see below how)**
+  Before and after:
+
+.. code-block:: yaml
+
+    # Before
+    run_something:
+      module.run:
+        - name: mymodule.something
+        - m_name: 'some name'
+        - kwargs: {
+          first_arg: 'one',
+          second_arg: 'two',
+          do_stuff: 'True'
+        }
+
+    # After
+    run_something:
+      module.run:
+        mymodule.something:
+          - name: some name
+          - first_arg: one
+          - second_arg: two
+          - do_stuff: True
+
+- Previous behaviour of the function :py:func:`module.run <salt.states.module.run>` is
+  still kept by default and can be bypassed in case you want to use behaviour above.
+  Please keep in mind that old implementation will be entirely removed in version "Oxygen".
+  In order to access new function behaviour, please add the following configuration to the
+  minion setup. Note that the configuration below you can also deploy separately to all
```

Comment:
The note here could also be put in a ``.. note::`` RST block.

I don't understand this note though, to be honest. Are you suggesting that all minions should have their config files edited? Have you considered using [config.get](https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.config.html#salt.modules.config.get) to get these configuration options? It checks grains and pillar as well as the minion config file. This would allow for this option to be turned on simply by editing the pillar data (or using ``grains.setval`` to set the value in the minion's /etc/salt/grains). Both of these options seem less intrusive IMO than editing all minions' config files.